### ISSUE TITLE
Re-train EQTransformer

### ISSUE THREAD

**Issue by babulab** [seisbench/seisbench#180]
Hi everyone, 

I would like to re train EQTransformer (my goal is decrease the number of trainable parameters) but I'm having trouble doing so. I'm guided by the [example](https://colab.research.google.com/github/seisbench/seisbench/blob/main/examples/03a_training_phasenet.ipynb) provided in the documentation, at first I had a little problem with the input dimension, because EQT uses 60s, not 30s like Phasenet, this problem was easily solved. Then I have another problem in the loss [function,](https://colab.research.google.com/github/seisbench/seisbench/blob/main/examples/03a_training_phasenet.ipynb#scrollTo=5ef5581f&line=1&uniqifier=1) because the _y_pred_ was a tuple, not a tensor, so the multiplication between _y_true_ and _y_pred_ was giving error, which I solved doing y_pred = torch.stack(y_pred, 1). Doing that didn't give any error, but the main problem is that I'm getting very bad result. I'm trying with the Iquique and ETHZ database, in both cases the results are poor.
I know that the model EQTransformer was already pre-trained with Iquique, I checked if there is more metadata in the json attached to each pre-trained model, but is not enough info :/.

If anyone can provide more information, ideally some code, showing how to re train EQT and if is possible include if you use data augmentation. I would really appreciate it.

Thanks in advance.

**Comment by yetinam** [comment]
Hi @babulab,

thanks for your question. As indicated in the notebook, there is a far more comprehensive code for training models available here: https://github.com/seisbench/pick-benchmark
Regarding the loss function for EQTransformer, it is defined [here](https://github.com/seisbench/pick-benchmark/blob/31057a1981f6cd50f828fbf8c284c2b0a228a502/benchmark/models.py#L404) and [here](https://github.com/seisbench/pick-benchmark/blob/31057a1981f6cd50f828fbf8c284c2b0a228a502/benchmark/models.py#L420). The augmentations we used are defined [here](https://github.com/seisbench/pick-benchmark/blob/31057a1981f6cd50f828fbf8c284c2b0a228a502/benchmark/models.py#L494).

For further explanations you might also want to check the similar previous issues #110 and #112.
